docs/tutorials/cloud_fit_deploy/autopilot-autogluon.md

```diff
@@ -0,0 +1,4 @@
+# New managed AutoGluon-Tabular experience on Amazon SageMaker Autopilot
+:label:`autopilot_autogluon`
+
+[Checkout managed AutoGluon-Tabular experience on Amazon SageMaker Autopilot](https://aws.amazon.com/blogs/machine-learning/amazon-sagemaker-autopilot-is-up-to-eight-times-faster-with-new-ensemble-training-mode-powered-by-autogluon/)
```

Comment:
Add a brief 4-5 sentence overview of what AutoPilot is, maybe just copy-paste from their own page, just to give some context to the user.

Reply:
Some statement found on their [website](https://aws.amazon.com/sagemaker/autopilot/?sagemaker-data-wrangler-whats-new.sort-by=item.additionalFields.postDateTime&sagemaker-data-wrangler-whats-new.sort-order=desc):
```
Amazon SageMaker Autopilot eliminates the heavy lifting of building ML models. You simply provide a tabular dataset and select the target column to predict, and SageMaker Autopilot will automatically explore different solutions to find the best model. You then can directly deploy the model to production with just one click or iterate on the recommended solutions to further improve the model quality.
```